source/euler/hyperbolic_system.h

```diff
@@ -649,6 +664,17 @@ namespace ryujin
         state_type apply_galilei_transform(const state_type &state,
                                            const Lambda &lambda) const;
         //@}
+
+
+        /**
+         * system dependent postprocessing step @p U is our solution state
+         * and we will define work to do on this which is Hyperbolic System Dependent
```

Comment:
Write complete sentences with correct grammar, upper/lowercase, and a period at the end. Remember that you're documenting *for a reader*.